server/models/CommentReaction.js

```diff
@@ -0,0 +1,84 @@
+import { EMOJI_TYPES } from '../constants/emojiTypes';
+import models from './index';
+
+export default function (Sequelize, DataTypes) {
+  const CommentReaction = Sequelize.define(
+    'CommentReaction',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+
+      UserId: {
+        type: DataTypes.INTEGER,
+        references: { key: 'id', model: 'Users' },
+        onDelete: 'NO ACTION',
+        onUpdate: 'CASCADE',
+        allowNull: false,
+      },
+
```

Comment:
Let's add it here too

```suggestion
      FromCollectiveId: {
        type: DataTypes.INTEGER,
        references: { key: 'id', model: 'Users' },
        onDelete: 'CASCADE',
        onUpdate: 'CASCADE',
        allowNull: false,
      },
```